eea/facetednavigation/widgets/autocomplete/configure.zcml

```diff
@@ -20,10 +20,11 @@
     file="./view.css"
    />
 
-  <browser:page
+  <browser:view
     for="*"
-    name="autocomplete-suggest"
-    class=".widget.AutocompleteSuggest"
+    provides="eea.faceted.vocabularies.autocomplete.IAutocompleteSuggest"
+    name="solr-autocomplete-suggest"
```

Comment:
As you changed the name of this view, I think an upgrade step is also required.

1. Find all object that are IFacetedNavigableEnabled
2. See if autocomplete widget is present
3. Set **autocomplete_view** to **solr-autocomplete-suggest**
Example: https://github.com/collective/eea.facetednavigation/blob/master/eea/facetednavigation/upgrades/evolve71.py